### DISCUSSION TITLE
Welcome to faker.js Discussions!

### DISCUSSION THREAD

**Discussion by damienwebdev** [faker-js/faker#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bchavez** [comment]
Hey. I'm the maintainer for [**Bogus**](https://github.com/bchavez/Bogus) - a port (and _superset_) of `faker.js` for the C#/.NET community.

The underlying locale data for **Bogus** uses `faker.js` to ensure we are getting the latest community contributions. **Bogus** now uses this upstream repo for pulling down updates to our locales: https://github.com/bchavez/Bogus/commit/321341a137a7f46e09bbabff78c3215f6c6c6ffa.

There's probably not much I can do to help from a JS perspective, but feel free to let me know.

Also, one tip, we have UI Faces (`internet.avatar()`) hosted on IPFS that's been working pretty well for **Bogus**. A long time ago, someone removed the original AWS S3 bucket for UI Faces and caused a bunch of 404s in `faker.js` and **Bogus**. I was able to recover many of the UI Faces from Internet Archive, and we now have them in IPFS hosted over [Cloudflare's IPFS gateway](https://www.cloudflare.com/distributed-web-gateway/). You can find all the details here: https://github.com/bchavez/Bogus/issues/336. Anyone is free and welcome to help seed & host these files on IPFS; we also have them pinned on various IPFS pinning services.

Thanks,
Brian
